src/test/GameWorldWithLuaAccess.h

```diff
@@ -33,18 +33,23 @@
 #include "libutil/Log.h"
 #include "libutil/StringStreamWriter.h"
 #include "libutil/colors.h"
+#include "Game.h"
 #include <boost/test/unit_test.hpp>
+#include <boost/weak_ptr.hpp>
+#include <boost/shared_ptr.hpp>
+#include <boost/make_shared.hpp>
 #include <vector>
 
-class GameWorldWithLuaAccess : public GameWorldGame
+class GameWithLuaAccess : public Game
```

Comment:
Idea: Replace this by a simple function that creates a `shared_ptr<Game>`, creates the lua in it and return it.
The fixtures need to store a shared_ptr then and call this function to init it.